ts/common/helpers/uihelper.js

```diff
@@ -0,0 +1,33 @@
+System.register(["../promise"], function (exports_1, context_1) {
+    "use strict";
```

Comment:
we do not need to commit compiled files into git as they can be generated again if need